docs/server-workflow-search.md

```diff
@@ -109,184 +110,63 @@ There are some pre-allowlisted search attributes that are handy for testing:
 - CustomDatetimeField
 - CustomStringField
 
-Their types are indicated in their names.
-
-### Value Types
-
-Here are the Search Attribute value types and their correspondent Golang types:
-
-- Keyword = string
-- Int = int64
-- Double = float64
-- Bool = bool
-- Datetime = time.Time
-- String = string
-
-### Limit
-
-We recommend limiting the number of Elasticsearch indexes by enforcing limits on the following:
-
-- Number of keys: 100 per Workflow
-- Size of value: 2kb per value
-- Total size of key and values: 40kb per Workflow
+When performing a [ContinueAsNew](/docs/go-continue-as-new/) or using [Cron](/docs/go-distributed-cron/), search attributes will be carried over to the new run by default.
 
-Temporal reserves keys like NamespaceId, WorkflowId, and RunId. These can only be used in list queries. The values are not updatable.
+## Search queries
 
-### Upsert Search Attributes in Workflow
+You can perform search queries for Workflows using an SQL-like "Where" clause via the [CLI](/docs/tctl/#search-workflows), list APIs via one of the SDKs([Go](/docs/go-search-apis)), or via the Temporal Web UI.
 
-[UpsertSearchAttributes](https://pkg.go.dev/go.temporal.io/sdk/workflow#UpsertSearchAttributes) is used to add or update search attributes from within the Workflow code.
+In the Temporal Web UI, use the "Basic/Advanced" button to switch to "Advanced" mode and type the query into the search box.
 
-Go samples for search attributes can be found at [github.com/temporalio/temporal-go-samples](https://github.com/temporalio/temporal-go-samples/tree/master/searchattributes).
+The following criteria applies to search queries:
 
-UpsertSearchAttributes will merge attributes to the existing map in the Workflow. Consider this example Workflow code:
+- These operators are supported in search queries:
+  - **AND, OR, ()**
+  - **=, !=, >, >=, <, <=**
+  - **IN**
+  - **BETWEEN ... AND**
+  - **ORDER BY**
+- You will only be able to search for Workflows from one Namespace at a time.
+- The default pagesize is 1000, and cannot be larger than 10,000.
+- The range on a Temporal timestamp search query (StartTime, CloseTime, ExecutionTime) cannot be larger than 9223372036854775807 (maxInt64 - 1001)
+- Search queries using a time range will have a 1ms resolution.
+- Search query column names are case sensitive.
+- A `ListWorkflow` search query may take longer when retrieving a large number of Workflows (10M+).
+- To retrieve a large number of Workflows without ordering, use the `ScanWorkflow` API.
+- To efficiently count the number of Workflows, use the `CountWorkflow` API
+- To list only open Workflows, add `CloseTime = missing` to the end of the query.
```

Comment:
This seems like a very specific bullet for this list.

Reply:
Ya, a bit - but also sort of a weird call to have to make some place else.